### DISCUSSION TITLE
1.16.5 - Nightly: fixed fluid and block updates, new fluidloggable config list

### DISCUSSION THREAD

**Discussion by Felicis** [Felicis/Water-Physics-Overhaul#50] [Nightly Releases]
THIS VERSION IS NIGHTLY: backup your worlds, expect crashes! Do not post issues for this version, instead join the discussion.

- completely overhauled fluid and block updates
  - anvil falls correctly
  - multi-blocks (door, bed) build correctly
  - placing sign in water creates pop-up
  - fixed no sounds and animations when placing blocks in survival
- new fluidlogging configs
  - configs reload automatically
  - default behaviour config (vanilla vs overhauled)
  - custom override lists (whitelist wins over blacklist)
    - **fluidloggable** blocks whitelist and blacklist
    - blocks **destroyed by fluids** whitelist and blacklist

<hr /><em>This discussion was created from the release <a href='https://github.com/Felicis/Water-Physics-Overhaul/releases/tag/0.1.3-alpha5'>1.16.5 - Nightly: fixed fluid and block updates, new fluidloggable config list</a>.</em>

**Comment by wglpython** [comment]
So, will this mod consider compatibility with the fresh water and salt water fluids from TFC:TNG?

**Reply by Felicis** [reply]
That's the plan 😁 but I haven't tested mod compatibility yet. Remind me again once we're testing mod compatibility!
